### DISCUSSION TITLE
How to display OpenThread Border Router version?

### DISCUSSION THREAD

**Discussion by olavt** [openthread/openthread#10582] [Q&A]
I often see references to Thread 1.3.0 etc.

How can I display the OTBR version number?

sudo ot-ctl thread version returns just "4"

and

sudo ot-ctl version returns: "OPENTHREAD/4e3483c; POSIX; Jul 23 2024 18:36:29"

Also, how can I check the Thread version number for my Apple TV?

**Comment by wgtdkp** [comment]
You can find out the printable Thread version in Thread Border Router mDNS service (the "_meshcop._udp") service.

You can use a mDNS service browser app (e.g. https://play.google.com/store/apps/details?id=com.druk.servicebrowser&hl=en_US&pli=1) to do this. There is a "tv" TXT field in discovered Border Router services.

![8EkPvqpX4r49phh](https://github.com/user-attachments/assets/9bd8dc89-317a-489b-b6f1-be5aab584108)

**Reply by olavt** [reply]
Thanks! I got the information using this command:

avahi-browse -r -t _meshcop._udp
